docs/tutorials/facebook-api.md

```diff
@@ -248,7 +430,7 @@ FB.api(path, function (lists) {
             app.assets.load(asset);
 
             asset.ready(function (asset) {
-                self.createPhoto(asset.resource);
+                this.createPhoto(asset.resource);
```

Comment:
If you're going to use `this` here, shouldn't the enclosing function be an arrow function?